v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# SPLUNK_FORWARD_SERVER shoudl be delcared as a config in the setup-credentials util
+
+source /etc/environment
+
+etcdctl set /splunk/SPLUNK_FORWARD_SERVER $SPLUNK_FORWARD_SERVER
```

Comment:
@chr0n1x That's what I hope we can extend the secrets file to be, but atm it doesn't work for environment variables. I am going to start parsing our userdata and mesos-systemd scripts to come up with the best way to approach this. For right now, if `$SPLUNK_FORWARD_SERVER` is needed, it'd have to be set in userdata.

@ts3ng instead of doing
```
etcdctl set /splunk/SPLUNK_FORWARD_SERVER $SPLUNK_FORWARD_SERVER
```
you could make that a config in `secrets.json`. So just remove that line 6 and in `secrets.json`, add:
```
"/splunk/SPLUNK_FORWARD_SERVER": "value",
```
to the "configs" section.